### DISCUSSION TITLE
How to submit the code?

### DISCUSSION THREAD

**Discussion by ArkaPrabhaChowdhury** [open-sauced-craftwork/browser-extension#39] [Q&A] [answered]
Where and how do I create a pull request? @bdougie

**Comment by bdougie** [comment, marked as answer]
We answered this here and a few other discussions. https://github.com/open-sauced-craftwork/browser-extension/discussions/8

You should create a new branch for your work in your repo.

**Reply by ArkaPrabhaChowdhury** [reply]
> We answered this here and a few other discussions. https://github.com/open-sauced-craftwork/browser-extension/discussions/8
> 
> You should create a new branch for your work in your repo.

@bdougie I have done that already in the private repo. Is that it?

**Reply by anush008** [reply]
Yes. A PR to merge the branch you are working on to the main branch.
